go-apps/meep-metrics-engine/api/swagger-v2.yaml

```diff
@@ -0,0 +1,790 @@
+        items:
+          $ref: "#/definitions/EventValue"
+    description: "Event metrics query response"
+    example:
+      columns:
+      - "event"
+      values:
+      - description: "Mobility event ue1 to poa3"
+        time: "2019-11-24T12:45:00-5:00"
+      - description: "Mobility event ue1 to poa2"
+        time: "2019-11-24T12:44:00-5:00"
+      name: "event metrics"
+  EventSubscriptionParams:
+    type: "object"
+    properties:
+      clientCorrelator:
+        type: "string"
+        example: "83"
+        description: "Uniquely identifies this create subscription request. If there\
+          \ is a communication failure during the request, using the same clientCorrelator\
+          \ when retrying the request allows the operator to avoid creating a duplicate\
+          \ subscription."
+      callbackReference:
+        $ref: "#/definitions/Events_callbackReference"
+      eventQueryParams:
+        $ref: "#/definitions/EventQueryParams"
+      recurrence:
+        type: "integer"
+        example: 1
+        description: "Recurrence of recurring-time-based notifications"
+      subscriptionType:
+        type: "string"
+        example: "Recurring-time-based"
+        description: "Type of subscription triggering notifications"
+        enum:
+        - "Recurring-time-based"
+    description: "Events subscription parameters"
+    example:
+      clientCorrelator: "0123"
+      callbackReference:
+        notifyURL: "http://clientApp.example.com/metric_event_notifications/123456"
+      eventQueryParams:
+        tags:
+        - name: "type"
+          value: "MOBILITY"
+        fields:
+        - "event"
+      recurrence: 1
+      subscriptionType: "Recurring-time-based"
+  EventSubscriptionResponse:
+    type: "object"
+    properties:
+      subscriptionId:
+        type: "string"
+        example: "1.2556487E7"
+        description: "Subscription identifier"
+      clientCorrelator:
+        type: "string"
+        example: "83"
+        description: "Uniquely identifies this create subscription request. If there\
+          \ is a communication failure during the request, using the same clientCorrelator\
+          \ when retrying the request allows the operator to avoid creating a duplicate\
+          \ subscription."
+      callbackReference:
+        $ref: "#/definitions/Events_callbackReference"
+      resourceURL:
+        type: "string"
+        format: "uri"
+        example: "http://localhost:8291/v2/subscriptions/events/subscription123"
+        description: "Self referring URL."
+      eventQueryParams:
+        $ref: "#/definitions/EventQueryParams"
+      recurrence:
+        type: "integer"
+        example: 1
+        description: "Recurrence of recurring-time-based notifications"
+      subscriptionType:
+        type: "string"
+        example: "Recurring-time-based"
+        description: "Type of subscription triggering notifications"
+        enum:
+        - "Recurring-time-based"
```

Comment:
Rename property: updateInterval

Description: "Notification interval in seconds, disabled if set to 0"

Reply:
fixed